e2e/fixtures/go/foo.go

```diff
@@ -1,15 +1,15 @@
 package master
 
-// @OctoLinkerResolve(https://golang.org/pkg/context)
+// @OctoLinkerResolve(https://pkg.go.dev/context)
 import "context"
 
-// @OctoLinkerResolve(https://golang.org/pkg/math/big)
+// @OctoLinkerResolve(https://pkg.go.dev/math/big)
 import "math/big"
 
-// @OctoLinkerResolve(https://golang.org/pkg/k8s.io/apimachinery/pkg/api/errors)
+// @OctoLinkerResolve(https://pkg.go.dev/k8s.io/apimachinery/pkg/api/errors)
 import "k8s.io/apimachinery/pkg/api/errors"
 
-// @OctoLinkerResolve(https://golang.org/pkg/k8s.io/api/core/v1)
+// @OctoLinkerResolve(https://pkg.go.dev/k8s.io/api/core/v1)
```

Comment:
@stefanbuck my initial change got the tests passing but now they're failing again. This url resolved to `https://github.com/kubernetes/api/tree/master/core/v1` yesterday but now it's resolving to `https://golang.org/pkg/k8s.io/api/core/v1` again. The tests won't pass with either url though. Any ideas what's going on here? Is the api caching the wrong url?

Reply:
> Is the api caching the wrong url?

Yes, I think this is what is happening here. In https://github.com/OctoLinker/api/pull/426/files#diff-90e29dc49ee3814a005e7dd6d34a3bd3bacbcc1daba54f195e21cbb70bca564aR27 I updated the caching of 404 ping results from 15min to 48 hours. Not sure how to resolve this issue best to be honest.

Reply:
btw for me it still resolves to `https://github.com/kubernetes/api/tree/master/core/v1`. The reason is because my location hits another Redis cache (we have a Redis cache running in four major regions). This makes it even more complicated. Maybe we should just merge it and get the new release out

Reply:
I just removed the revert I tried so the test now points to the github url, which is also what I get when testing with one of the api PR instances.

Does this change need any tests? https://github.com/OctoLinker/OctoLinker/pull/1536/files#diff-f1c4ee55132845e20cf1dd168feb8f34ed36e11d1651ecb94f107c059d7b3fa0

Reply:
fyi https://github.com/OctoLinker/api/pull/439

Reply:
The tests passed, better merge this before they start to fail again 😄

Reply:
Maybe GA are running in different regions, but yeah let's merge this